app/src/components/UserProfile.js

```diff
@@ -44,18 +85,18 @@ export default function UserProfile() {
               onChange={handleInputChange}
             />
           </label>
-          <label id="userLocation">
-            Location
+          <label id="zipCode">
+            Zip Code
             <br />
             <input
```

Comment:
Since you mentioned that inputting anything aside from numbers would break your app,

- you can do html validation to make sure only numbers are allowed to be entered and only with 5 digits
example here https://css-tricks.com/html-for-zip-codes/
- but I prefer input `type="number"` because on mobile it makes the keyboard into a number pad so numbers are more easily entered.

_100% optional_
to be safe (but not entirely necessary for the mvp), depending on where it breaks with the wrong zipcode ( in the database? then validate the zipcode again on express and return an error if it isn’t a valid zip code ) because technically someone can modify and bypass the front-end client